### PR TITLE
presenter for the cancel donation use case

### PR DESCRIPTION
**Pull request by KaiNissen**
This PR adds the presenter class for the cancel donation use case as well as corresponding route tests. It is part of [phab:123046](https://phabricator.wikimedia.org/T123046).

### CODE REVIEW

File: app/routes.php [hunk 1]
```diff
@@ -316,21 +316,12 @@ function( Request $request ) use ( $app, $ffFactory ) {
...
-				'status' => 'OK',
-				'message' => '',
-			] );
-		}
-
-		return $app->json( [
-			'status' => 'ERR',
-			'message' => 'TODO',
-		] );
+		return $ffFactory->newCancelDonationHtmlPresenter()->present(
```
Comment (JeroenDeDauw) [inline app/routes.php]: You can just pass the response to presenter.

**Comment by JeroenDeDauw** [comment]
Looks good +1. Probably good if someone also double checks the german stuffs

**Comment by KaiNissen** [comment]
rebased

File: app/templates/Donation_Cancellation_Confirmation.twig [hunk 2]
```diff
@@ -1,9 +1,19 @@
 <div id="greenBox" class="inhalt boxShadow">
 
 	<div class="text">
-		<h2>{$ 'donation-cancellation-header'|trans({}) $}</h2>
+		<h2>
+			{% if cancellationSuccessful %}
+			{$ 'donation-cancellation-header'|trans({}) $}</h2>
```
Comment (gbirke) [inline app/templates/Donation_Cancellation_Confirmation.twig]: You can do just `{$ 'donation-cancellation-header'|trans $}` when you have no variables.

**Pull request merged**